Benchmark/src/PWR003.cpp

```diff
@@ -0,0 +1,66 @@
+#include "Benchmark.h"
+
+// Forward-declare the functions to benchmark
+extern "C" {
+void example(const int N, const double *masses, double *weights);
+void solution(const int N, const double *masses, double *weights);
+void example_f(const int N, const double *masses, double *weights);
+void solution_f(const int N, const double *masses, double *weights);
+}
+
+// Size adjusted to fit execution on micro-seconds
+constexpr int N = 128 * 128 * 128;
+
+#if OCB_ENABLE_C
+
+static void CExampleBench(benchmark::State &state) {
+  const auto masses = OpenCatalog::CreateRandomVector<double>(N, 35.0, 80.0);
```

Comment:
Is there a particual reason you chose `35.0` and `80.0` to initialize the `masses`? If not, I would use the default `lower` and `upper` bounds in `CreateRandomVector`.

Reply:
No, I wanted to stick to "real" masses, but It doesn't has sense, so I'll delete them.